ideascube/monitoring/templates/monitoring/stock.html

```diff
@@ -15,7 +15,7 @@
         {% for item in module.objects  %}
             <tr class="stockitem" id="stockitem-{{ item.pk }}"><td>{{ item }} <em>{{ item.description|truncatewords:10 }}</em></td><td>{% trans 'Total:' %} {{ item.specimens.physical|length }}</td><td class="actions"><a href="{% url 'monitoring:stockitem_update' pk=item.pk %}" title="{% trans 'Edit' %}">{% fa 'pencil-square' %}</a> <a href="{% url 'monitoring:specimen_create' item_pk=item.pk %}" title="{% trans 'Add new specimen of this item' %}">{% fa 'plus-square' %}</a></td></tr>
             {% for specimen in item.specimens.physical %}
-                <tr class="specimen"><td>{{ specimen.barcode|default:'—' }}</td><td>{{ specimen.count }}</td><td class="actions"><a href="{% url 'monitoring:specimen_update' pk=specimen.pk %}" title="{% trans 'Edit this specimen' %}">{% fa 'pencil-square' %}</a></td></tr>
+                <tr class="specimen"><td>{{ specimen.pretty_identifier|default:'—' }}</td><td>{{ specimen.count }}</td><td class="actions"><a href="{% url 'monitoring:specimen_update' pk=specimen.pk %}" title="{% trans 'Edit this specimen' %}">{% fa 'pencil-square' %}</a></td></tr>
```

Comment:
This is not easy here. There is a table but the rows are shared between the item and the specimens.

- We could split the column (and span the item description cell on two columns). But we have to find nice header's text(s).
- We can just write in the cell something link "Barcode : {{ specimen.barcode|default:'—' }}, Serial number : {{ specimen.serial|default:'—' }}" But it would lead to a lot of duplication of "Barcode :" and "Serial : " and reduce readability.